ayekan/exporters/puppetdb-exporter-deployment.yaml

```diff
@@ -0,0 +1,206 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
A `namespace` should be set on all resources going forward as I'm planning to start migrating deployments into fleet starting this week but I don't want to hold this up any further so I'll mark the PR approved.